### DISCUSSION TITLE
Trigger new client connection on server and client sides

### DISCUSSION THREAD

**Discussion by newlaurent62** [jamulussoftware/jamulus#2154] [Ideas]
Hello,

Would be nice to be able to trigger a new client connection on server and/or client sides.

The idea is to generate a sound or a notification to the end user when a new client has connected. This sound should not be part of the multitrack server or stereo client recording !

I'm a linux user and i use Jack.

BR,
Laurent

**Comment by dcorson-ticino-com** [comment]
Why does it make a difference that everyone must hear if someone has connected?
I do not see the use case.

**Comment by newlaurent62** [comment]
If someone use "mute myself" when connecting, you can't hear that it has connected, right ?

Or if your server is running and you are not client connected to your server, you can't hear that someone has connected.

The (system or sound) notification generated when client has connected could be optional (on client or server side).
